hoot-core/src/main/cpp/hoot/core/conflate/highway/HighwayMatchCreator.cpp

```diff
@@ -95,7 +102,10 @@ class HighwayMatchVisitor : public ElementVisitor
     env->expandBy(getSearchRadius(e));
 
     // find other nearby candidates
-    set<ElementId> neighbors = _map->findElements(*env);
+    set<ElementId> neighbors = IndexElementsVisitor::findNeighbors(*env,
+                                                                   getIndex(),
+                                                                   _indexToEid,
+                                                                   getMap());
```

Comment:
I didn't instrument it to check. But I'm curious! If I come up with any numbers, I'll share them.